modules/main-rules/pages/assault-moves-and-charges.adoc

```diff
@@ -2,36 +2,39 @@
+== Assault moves
 
 IMPORTANT: For detachments that have Blast markers on them, you must pass a Leadership test before you can move them.
 
-* You don't have to move these units more than 5 cm, unless you want to.
-+
-(You may simply want to manoeuvre into range for a firefight or to grab an objective, for example.)
+Moves in the Assault phase are subject to all the rules we described for the Movement phase -- except that the Snap-fire rule does *not* apply in the Assault phase.
+
+In addition, when you move each detachment in the Assault phase, move each of its units one after the other. Each unit:
+
+* *Must* move at least 5cm, or into base contact with an enemy within 5 cm. Any movement beyond this is optional.
+* *Must* move as directly as possible towards an enemy unit, but that unit does not need to be the nearest enemy, or even in a Detachment that is being assaulted. The unit can detour around Dangerous terrain.
 +
-However, you may not move assaulting units *past* any enemy units footnote:[To avoid a lengthy explanation here we discuss this in more detail at +[TODO: FAQ -- Moving past in the Assault phase]+.] -- unless those enemies are already engaged in close combat with other friendly units.
-* You must move these units towards the enemy -- but not necessarily towards the _nearest_ enemy.
-* You can move these units less than 5 cm *only* where they charge into close combat (base contact) with enemy units that are within 5 cm.
+[NOTE.e40k]
+.Assault moves through Dangerous terrain
+====
+We have added the option for units to avoid Dangerous terrain during their assault moves.
+====
+* *Cannot* move away from the closest enemy unit.
+** The closest enemy unit is the one that can be reached by the shortest route.
+** Any enemy unit that already has a friendly unit in base contact can be ignored.
 
-Otherwise, moves in the Assault phase are subject to all the rules we described for the Movement phase -- except that the Snap-fire rule does not apply in the Assault phase.
+////
+[TODO IJW: I've put the war engine part within comment marks for now, and we can decide if it should be included here or only appear in the WE pages.]
 
-== Charge
+War engines follow the same rules for assault moves, but their own movement rules take priority. War engines can choose not to move in the Assault phase, but will not be able to initiate Close Combats or Firefights if they do not move.
+////
 
-As you move detachments in this phase, you may declare that any or all of them will *charge* so that they become engaged in close combat with the enemy.
+== Charge moves
 
-This is the only time that you may move your units into base contact with enemy units.
-And where you do move any individual unit into base contact with an enemy unit, we call this a *charge move*.
+If your unit's assault move brings it into base contact with an enemy unit, this is referred to as a *charge move*, or a *charge*.
```

Comment:
I'm making some edits to this topic at the moment, mostly just different rephrasing. However, I think that this line is slightly problematic.

You're quite reasonably trying to construct things in such a way that we make assault moves and then state which of these are charges by definition because the units reached base contact.

However, I think that there is a bit of a 'chicken and egg' situation here. That is, I don't think that this construction works well given the special charge Speed bonuses for infantry, cavalry, etcetera. You only get their enhanced Speed value if you **will** make base contact.

So it isn't that "your unit's assault move brings it into base contact" per se (though that can be true), it is that the sub-set of assault moves that we call the charge move that brings it into base contact — which usually amounts to the same thing, but not always because of the special requirements for the bonus Speed.

I think that I'll finish my current phrasing edits and push them, and then we should look at bits like this — it get's really confusing if we're both making independent edits and suggestions at the same time, while also trying to compare our working copies with previous iterations.

Reply:
Hang on, you go on to qualify it, so maybe it works out. Still, I might have to do the rephrasing in several iterations to wrap my head around it.

Reply:
> Hang on, you go on to qualify it, so maybe it works out. Still, I might have to do the rephrasing in several iterations to wrap my head around it.

Yeah, I forgot to add in a note asking you to check these bits in particular, as even with the later text it's still slightly back-to-front in the order in which its explained...

Reply:
I'll resolve this conversation — we know that it'll take a few passes to get through the whole topic and to thrash out the 'chicken and egg bit' in particular.